openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -306,6 +304,115 @@ def build_layout(project_name, asset_name):
             "jpg",
         )
 
+    # Setup cryptomatte
+    bpy.context.scene.view_layers[
+        "ViewLayer"
+    ].use_pass_cryptomatte_asset = True
+
+    # Load base nodegroup and make it publishable
+    _compo_container, compo_datablocks = load_subset(
+        project_name,
+        "CompositingNodesBank",
+        "nodegroupMatteColorCorrect",
+        "Append",
+    )
+
+    input_image_node = None
+    for container in containers:
+        if container.get("avalon", {}).get("family") == "rig":
+            input_image_node = setup_character_compositing(
+                asset_name,
+                container.name,
+                list(compo_datablocks)[0],
+                input_image_node,
+            )
+    else:
+        # Link last matte color correct node to composite node
+        composite_node = bpy.context.scene.node_tree.nodes["Composite"]
+        bpy.context.scene.node_tree.links.new(
+            input_image_node.outputs["Image"],
+            composite_node.inputs["Image"],
+        )
+
+
+def setup_character_compositing(
+    asset_name, character_name, source_compositing_nodegroup, input_image_node
```

Comment:
```suggestion
    asset_name: str,
    character_name: str,
    source_compositing_nodegroup: bpy.types.NodeTree,
    input_image_node: bpy.types.Node,
```
Can you add types to the argument list?